src/Analyzer/PatternString.php

```diff
@@ -19,8 +19,9 @@ public function matches(string $pattern): bool
             return false;
         }
 
-        if (!$this->containsWildcard($pattern) && str_starts_with($this->value, $pattern)) {
-            return true;
+        if (!$this->containsWildcard($pattern)) {
+            $slashedTerminatedPattern = str_ends_with($pattern, '\\')? $pattern : $pattern.'\\';
```

Comment:
Can we simplify these lines to explicit well the meaning?

Reply:
It's better now?

Reply:
Yes, much better @fain182